README.md

```diff
@@ -1,133 +1,29 @@
-<h1 align="center">Moai UI Kit 🗿</h1>
+# Moai UI Kit 🗿
 
-A React component library, where buttons look like buttons.
+A React Component Library, Where Buttons Look Like Buttons.
 
--   Website: [moaijs.com](https://moaijs.com)
--   Document: [docs.moaijs.com](https://docs.moaijs.com)
-
-![Moai](https://user-images.githubusercontent.com/30283022/117471857-7a748d00-af82-11eb-9fa1-cdb22db1ec16.png)
-
-# Install
-
-The [@moai/core] package is the basis of the Moai kit. It provides all basic
-components, from buttons to dialogs and tables, along with necessary CSS.
-Install it with your package manager:
-
-```
-yarn add @moai/core
-```
-
--   It has [react] and [react-dom] as peer dependencies, so make sure they are
-    installed. (Which is usually the case if you use a framework like [Next.js] or
-    [CRA].)
--   Moai is written entirely in TypeScript (strict mode!) so all typings are
-    bundled in the packages.
-
-[typing]: https://www.typescriptlang.org/docs/handbook/declaration-files/introduction.html
-[react]: https://www.npmjs.com/package/react
-[react-dom]: https://www.npmjs.com/package/react-dom
-[@moai/core]: https://www.npmjs.com/package/@moai/core
-[next.js]: https://nextjs.org/
-[cra]: https://create-react-app.dev/
-
-# Setup
-
-There are a few things you need to do **once** before using Moai's components:
-
-## Styling
-
-Import Moai's CSS bundle at your app root. This contains CSS for all components
-in the "core" package.
-
-```tsx
-import "@moai/core/dist/bundle.css";
-```
-
--   In Next.js, use the [custom App]. In CRA, use `src/index`.
--   The bundled CSS contains a [simple reset], so it's best to not include
-    another one.
--   All CSS for components use [CSS Modules]. This means all class names are
-    scoped so you can use other class-based libraries like [Tailwind CSS] or
-    [Tachyons] without worrying about naming conflict.
-
-[simple reset]: https://github.com/moaijs/moai/blob/main/core/src/global.css
-[css modules]: https://github.com/css-modules/css-modules
-[tailwind css]: https://tailwindcss.com
-[tachyons]: https://tachyons.io/
-[custom app]: https://nextjs.org/docs/advanced-features/custom-app
-
-## Theme
-
-Moai comes with both light and dark themes out of the box. However, there is no
-default, so you must explicitly set a class on your `html` tag to specify the
-theme: either `light` or `dark`.
-
--   To avoid a [FOUC], ensure that your `html` tag has the class in the initial
-    render. This is usually done via a "template" file. In CRA, for example, set it
-    in [`public/index.html`].
--   In Next.js, we recommend the [`next-themes`] package. It allows your users
-    to change the theme, including a "system" option, without any flashing in
-    initial render.
+![Moai preview](https://user-images.githubusercontent.com/5953369/123528873-5b3de480-d715-11eb-8c02-50a04f2d83f9.png)
```

Comment:
I just really not enjoy this preview image, the default macOS screenshot shadow too strong... Recommend remove the shadow or just took UI kits.

Reply:
Good suggestion, I'll take another one with my wallpaper